drivers/usb/device/usb_dc_sam0.c

```diff
@@ -234,7 +242,9 @@ int usb_dc_attach(void)
 	(void)memset(data->descriptors, 0, sizeof(data->descriptors));
 	regs->DESCADD.reg = (uintptr_t)&data->descriptors[0];
 
-	regs->INTENSET.reg = USB_DEVICE_INTENSET_EORST;
+	regs->INTENSET.reg = USB_DEVICE_INTENSET_EORST |
+			     USB_DEVICE_INTENSET_EORSM |
```

Comment:
Also for normal "SUSPEND," we should be handling WAKEUP, not EORSM. That is exactly what ASF does too.

One caveat is that all disabled interrupts seem to be automatically "handled" (acknowledged), so leaving WAKEUP disabled does not have any lasting effect (so it seems).